e2e/pages/adf/filePreviewPage.ts

```diff
@@ -286,15 +285,14 @@ export class FilePreviewPage {
 
         this.clickZoomOut();
 
-        this.checkCanvasWidth().then(function (width) {
+        this.checkCanvasWidth().then((width) => {
             zoomedOutWidth = width;
             if (actualWidth && zoomedOutWidth) {
                 expect(zoomedOutWidth).toBeLessThan(actualWidth);
             }
         });
 
-        this.checkCanvasHeight().then(function (height) {
-            zoomedInHeight = height;
+        this.checkCanvasHeight().then(() => {
```

Comment:
I'm pretty sure the fix in this case (`'zoomedInHeight' is declared but its value is never read.`) should have been just to replace the `zoomedInHeight = height;` with `zoomedOutHeight = height;`